WhoIsThat/WhoIsThat/Connections/CloudStorageService.cs

```diff
@@ -29,5 +29,15 @@ public static async Task<CloudBlockBlob> SaveBlockBlob(MediaFile file, string bl
 
             return blockBlob;
         }
+
+        public static string GetImageUri(string title)
+        {
+            var containerName = "images";
```

Comment:
You could probably have constants/config file for that

Reply:
Yeah, I guess I'm too lazy, but I really need to have one.